fdbserver/IKeyValueStore.h

```diff
@@ -24,6 +24,23 @@
 
 #include "fdbclient/FDBTypes.h"
 #include "fdbserver/Knobs.h"
+#include "flow/Error.h"
+#include "fdbclient/StorageServerInterface.h"
```

Comment:
This is a very weird dependency. IMO the checkpoint-specific stuff should live in a separate header from StorageServerInterface, since it also kind of pollutes that file with a ton of rocks-db specific stuff that also feels very abstraction-breaking.

Reply:
Removed.